calico-cloud/visibility/alerts.mdx

```diff
@@ -26,7 +26,13 @@ for common tasks that you can rename and edit to suit your own needs.
 
 **Recommended**
 
-We recommend turning down the aggregation level for flow logs to ensure that you see pod-specific results. {{prodname}} aggregates flow logs over the external IPs for allowed traffic, and alert events will not provide pod-specific results (unless the traffic is denied by policy).
+We recommend turning down the aggregation level for flow logs to ensure that you see pod-specific results. {{prodname}} aggregates flow logs over the external IPs for allowed traffic, and alert events will not provide pod-specific results (unless the traffic is denied by policy). 
+
+:::caution
+
+Turning down aggregation levels for flow logs increases the amount of data and may increase your bill.
```

Comment:
'increases the amount of data'

I'm not sure how the billing works. But it might be good to specify what's going on with this data, if we know. Data stored? Data transferred? Data something else?

Please feel free to disregard entirely if you see fit. LGTM!